ouroboros-consensus/test-storage/Test/Ouroboros/Storage/ChainDB/StateMachine.hs

```diff
@@ -287,9 +288,10 @@ run :: forall m blk. (IOLike m, HasHeader blk)
     -> ChainDB.Internal m blk
     -> ResourceRegistry m
     -> StrictTVar m SlotNo
-    ->    Cmd     blk (Iterator m blk (AllComponents blk)) (Reader m blk (AllComponents blk))
-    -> m (Success blk (Iterator m blk (AllComponents blk)) (Reader m blk (AllComponents blk)))
-run ChainDB{..} internal registry varCurSlot = \case
+    -> StrictTVar m Id
+    ->    Cmd     blk (WithEq (Iterator m blk (AllComponents blk))) (WithEq (Reader m blk (AllComponents blk)))
```

Comment:
Create a type alias for `WithEq (Iterator m blk (AllComponents blk))` (same for `Reader`).